Tests/Fixtures/TestProject/Project.xcodeproj/xcshareddata/xcschemes/App_Clip.xcscheme

```diff
@@ -4,7 +4,8 @@
    version = "1.3">
    <BuildAction
       parallelizeBuildables = "YES"
-      buildImplicitDependencies = "YES">
+      buildImplicitDependencies = "YES"
+      runPostActionsOnFailure = "NO">
```

Comment:
[memo] 
This is added from https://github.com/tuist/tuist/pull/2752 . It looks this is undocumented variable and it looks default is NO. We also can support this value later.

https://github.com/spotify/XCLogParser/issues/77#issuecomment-623661404

Reply:
opened: https://github.com/yonaskolb/XcodeGen/pull/1073